test/any_scheduler_test.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <unifex/any_scheduler.hpp>
+
+#include <unifex/inline_scheduler.hpp>
+#include <unifex/single_thread_context.hpp>
+#include <unifex/transform.hpp>
+#include <unifex/sync_wait.hpp>
+
+#include <gtest/gtest.h>
+
+using namespace unifex;
+
+TEST(AnySchedulerTest, SatisfiedConcept) {
+  EXPECT_TRUE(scheduler<any_scheduler>);
```

Comment:
This could be a static_assert

Reply:
The test fails either way, and I prefer the gtest diagnostic.